conf/machine/include/hmx-base.conf

```diff
@@ -10,7 +10,12 @@ BBMASK += "\
 meta-hostmobility-bsp/recipes-bsp/libubootenv \
 meta-mobility-poky-distro/recipes-connectivity/networkmanager/networkmanager_%.bbappend \
 "
+
+# Use packagegroup for hm-interface-names ?
```

Comment:
not needed but maybe in the future, if we have many machines with diffrent need of interface renaming